sparse_strips/vello_hybrid/src/render.rs

```diff
@@ -306,25 +323,44 @@ impl Renderer {
                         },
                         wgpu::BindGroupEntry {
                             binding: 1,
-                            resource: config_buf.as_entire_binding(),
+                            resource: config_buffer.as_entire_binding(),
                         },
                     ],
                 });
-                (alphas_texture, render_bind_group)
+                (alphas_texture, render_bind_group, config_buffer)
             } else {
                 let resources = self.resources.as_ref().unwrap();
                 (
                     resources.alphas_texture.clone(),
                     resources.render_bind_group.clone(),
+                    resources.config_buffer.clone(),
                 )
             };
             self.resources = Some(GpuResources {
                 strips_buffer,
                 alphas_texture,
                 render_bind_group,
+                config_buffer,
             });
         };
 
+        if dimensions_changed {
```

Comment:
Does this path still trigger if the config buffer was recreated?

Reply:
Check the comment above for more details.

Fixed in 0e5a4d09